project-base/storefront/components/Pages/Contact/ContactContent.tsx

```
@@ -51,16 +49,17 @@ export const ContactContent: FC = () => {
-                {settingsData?.settings?.contactFormMainText && (
-                    <div
-                        dangerouslySetInnerHTML={{ __html: settingsData.settings.contactFormMainText }}
-                        id="contact-form-description"
-                    />
-                )}
+                <PageHero
+                    icon={MailSecondaryIcon}
+                    title={t('Write to us')}
+                    description={
+                        settingsData?.settings?.contactFormMainText ? (
+                            <span dangerouslySetInnerHTML={{ __html: settingsData.settings.contactFormMainText }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)